examples/speaker_tasks/diarization/conf/inference/diar_infer_meeting.yaml

```diff
@@ -24,7 +24,7 @@ diarizer:
     parameters: # Tuned parameters for CH109 (using the 11 multi-speaker sessions as dev set) 
       window_length_in_sec: 0.63  # Window length in sec for VAD context input 
       shift_length_in_sec: 0.01 # Shift length in sec for generate frame level VAD prediction
-      smoothing: False # False or type of smoothing method (eg: median)
+      smoothing: median # False or type of smoothing method (eg: median)
```

Comment:
temp fix here is not a good idea in my mind. Could we quick patch it in code?

Reply:
Updated